azure/templates/post-deploy.yml

```diff
@@ -68,6 +68,35 @@ steps:
     workingDirectory: "$(Pipeline.Workspace)/s/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)"
     retryCountOnTaskFailure: 2
 
+  - bash: |
+      set -ex
+      if ! [[ $APIGEE_ENVIRONMENT =~ .*-*sandbox ]]; then
+        counter=0
+        base_path="$SERVICE_BASE_PATH"
+        apigee_env="$ENVIRONMENT"
```

Comment:
Shouldn't this be `APIGEE_ENVIRONMENT`? What's the difference between `APIGEE_ENVIRONMENT` and `ENVIRONMENT`?